_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,210 @@
+
+**More information**
+
+- [Release notes for feefilter][release notes feefilter]
+
+- [BIP133: `feefilter`
+  message](https://github.com/bitcoin/bips/blob/master/bip-0133.mediawiki)
+
+
+## BIP32 HD wallet support
+
+When Bitcoin Core starts for the first time, it will now generate a BIP32 Hierarchical Deterministic (HD) wallet where every private key in the wallet is derived from a single piece of information using a repeatable (deterministic) process. This means backing up that single piece of information will back up every private key your wallet will ever generate, ensuring that you can recover any bitcoins controlled by those private keys in the future.
+
+Backups are hard to get right, so please note the following information:
+
+- If you upgrade from any version of Bitcoin Core before 0.13.0, you will continue using the old style of wallet where each private key is generated individually, with (by default) up to 100 of them pre-generated in advance to make backups easier---meaning you need to create additional backups every 100 transactions, since each default-style transaction uses one private key.
+
+- If you create a new wallet with 0.13.0 (or above) and you change from the default unencrypted wallet to an encrypted wallet, a new HD wallet will be generated for you.  You will still have access to any bitcoins sent to the unencrypted wallet, but you will need to backup the wallet again.
+
+If you aren't sure whether you're using an HD wallet, you can check using the `getwalletinfo` RPC:
+
+- If you use the Bitcoin Core graphical user interface, you can click the *Help* menu, choose the *Debug* option, click the *Console* tab, and then type in `getwalletinfo`.
+
+- If you use the `bitcoin-cli` command to access the RPC interface, you can type `bitcoin-cli getwalletinfo`.
+
+In either case, if you see a line labeled "masterkeyid", then you're using an HD wallet; if you don't see it, then you're using a wallet with individually generated keys.
+
+Backing up an HD wallet ensures that you will be able to re-generate any private keys produced by that wallet in the future, but that is the only information you can recover from the backup in the future.  Any additional information you enter into your wallet after you make the backup, such as descriptions of transactions you sent or received, will be lost if you have to restore from the HD wallet backup, so we recommend that you continue to make regular backups of your wallet in order to preserve that information.
+
+Importantly, if you manually import any private keys to your wallet, they cannot be recovered using any backups made prior to the import, so you will need to make a new wallet backup and use that.
+
+**More information**
+
+- [Release notes][release notes hd wallet]
+
+- [Deterministic
+  wallets](https://en.bitcoin.it/wiki/Deterministic_wallet) (Bitcoin
+  Wiki)
+
+- [BIP32: Hierarchical Deterministic Wallets][BIP32]
+
+## Child pays for parent mining
```

Comment:
@sdaftuar I push commit ba8717a4bcb97b3581a5708c491ae9774e0da3a2 which I think fully addresses your concerns with a minimum of edits.  Thanks!